components/threading/global_threads/thread_list/thread_list.tsx

```diff
@@ -97,7 +110,21 @@ const ThreadList = ({
                 )}
             />
             <div className='threads'>
-                {children}
+                <VirtualizedThreadList
+                    key={`threads_list_${currentFilter}`}
+                    ids={currentFilter === ThreadFilter.unread ? unreadIds : ids}
+                    selectedThreadId={selectedThreadId}
+                />
+                {currentFilter === ThreadFilter.unread && !someUnread && isEmpty(unreadIds) ? (
+                    <NoResultsIndicator
```

Comment:
shouldn't this be either the threadlist or the noresults? why nesting the no result indicator inside a list that won't be populated?

Reply:
@Willyfrog it's the noresults for the unreads tab. So it's not like nesting in the list, but nesting in the left pane.
Would renaming the component from `thread_list` to `thread_list_pane` make it clearer?

PS: 
We have 3 "noresults" cases:
1. whole global threads page (when not following any thread)
2. right pane (when there is no selected thread)
3. left pane (threads list) unreads tab (when we have no unread threads) <--- the case we are discussing